src/checkqueue.h

```diff
@@ -130,6 +131,7 @@ class CCheckQueue
     explicit CCheckQueue(unsigned int batch_size, int worker_threads_num)
         : nBatchSize(batch_size)
     {
+        LogPrintf("Script verification uses %d additional threads\n", worker_threads_num);
```

Comment:
Seems deliberate to not clamp here instead, could you explain the reason?

Reply:
We already control the batch size from the chainman, so seemed logical to me to control the number of threads too.

Reply:
Do we expect [every call-site](https://github.com/bitcoin/bitcoin/pull/31313#discussion_r1846266844) to do the clamping themselves?

Reply:
I don't think this really matters, but seems consistent to control the arguments in the same place, just like with the batch size.